common-libs/user-management/src/dto.ts

```diff
@@ -0,0 +1,31 @@
+import { IsOptional, IsNumber, IsString } from 'class-validator'
```

Comment:
But we are not actually doing any validation here? So no need to depend on class-validator.

Reply:
It's copy-pasted from the original code. Should we replace/remove this?

Reply:
We're (slightly) modifying code in this MR anyway, so there is no reason to carry this unneeded dependency into the new package.